workspace/teamdata/generate_report.py

```diff
@@ -131,16 +160,16 @@ def get_project_cards(project_id):
 
 def get_slack_username(github_username):
     return {
-        "CarolineMorton": "@Caroline",
-        "Jongmassey": "@Jon",
-        "StevenMaude": "@Steve",
-        "evansd": "@dave",
-        "iaindillingham": "@Iain",
-        "inglesp": "@inglesp",
-        "milanwiedemann": "@Milan",
-        "rebkwok": "@Becky S",
-        "robinyjpark": "@Robin",
-    }.get(github_username) or github_username
+        "CarolineMorton": "<@Caroline>",
```

Comment:
These should really be updated to use slack user ids so it doesn't break if someone changes their slack display name.  But another time.